docs/content/posts/clase12.md

```diff
@@ -72,18 +46,12 @@ Several notebooks with analysis examples and an interface to launch the framewor
 
 ## Recorded Class
 
-### as a YouTube video
-
 <CENTER>
 
 <iframe width="100%" height="500" src="https://www.youtube.com/embed/ptbiB_0bUf4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 </CENTER>
```

Comment:
comment this for the time being to not confuse participants?

Reply:
Right, let me do it again